test/testtokenize.cpp

```diff
@@ -3129,6 +3129,11 @@ class TestTokenizer : public TestFixture {
                              "    { }\n"
                              "}");
         ASSERT_EQUALS("", errout.str());
+
+        // #8333
+        ASSERT_THROW(tokenizeAndStringify("struct A { int f(const); };"), InternalError);
```

Comment:
Sorry but I was only talking about the last line of code. I would prefer that the tests are moved back to testgarbage. A comment on the last test case can mention that the code is fine and no exception should be thrown.

It is a corner case if it belongs in testgarbage or testtokenizer.

Reply:
Ok, I understand what you mean. Update in underway